hosts

```diff
@@ -1079,7 +1079,6 @@
 2404:6800:4005:80b::2003 certificate-transparency.org
 2404:6800:4005:80b::2003 www.certificate-transparency.org
 2404:6800:4008:c00::79 code.getmdl.io #ghs.l.google.com
-

```

Comment:
All snippets end with two blank line, but the first snippet `01_google.txt` in current version has only one, hence the merged hosts from snippets has a line missing here.